### DISCUSSION TITLE
Crossover Ambicioso

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#446] [Imagens]
Post de referência: https://www.vish.blog.br/imagens/crossover-ambicioso/

**Comment by maryon-telvanni** [comment]
Incrível que os três conseguiram estacionar feito babacas

**Comment by one-eyed-willy** [comment]
> [Maryon Telvanni](</perfil/?user=maryon-telvanni>) disse [segunda-feira, 4 de julho de 2022](<https://www.vish.blog.br/imagens/crossover-ambicioso/#comment_1174071904>):

> Incrível que os três conseguiram estacionar feito babacas

Não dá pra estacionar direitinho *if there's something strange in your neighborhood* :grin:
